driver/src/price_finding/price_finder_interface.rs

```diff
@@ -97,6 +106,7 @@ pub fn execute_private_solver(
         .arg(format!("{}{}", "/app/", input_file.to_owned()))
         .arg(format!("--outputDir={}{}", "/app/", result_folder))
         .args(&["--solverTimeLimit", &time_limit])
+        .arg(format!("--minAvgFeePerOrder={}", min_avg_fee_per_order))
```

Comment:
How unfortunate that these arguments differ slightly across our solvers/.

Reply:
The two solvers have different standards. Open-solver takes all parameters-like-this, while the standard solver takes all parametersLikeThis.

Hence, we would have to switch all parameters in one solver. Maybe we do that in the future.